content/ember-data/v2/record-lifecycle-event-methods.md

```diff
@@ -0,0 +1,30 @@
+---
+id: record-lifecycle-event-methods
+title: Record Lifecycle Event Methods
+until: '4.0.0'
+since: '3.12.0-canary.2'
+---
+## Deprecating Record Lifecycle Event Methods
+As described in ['RFC 0329'](https://github.com/emberjs/rfcs/pull/329) -
+>...the following lifecycle methods on DS.Model will also be deprecated: becameError, becameInvalid, didCreate, didDelete, didLoad, didUpdate, ready, rolledBack.
+>
+> When a model is instantiated for the first time with any of these methods a deprecation warning will be logged notifiying the user that this method will be deprecated and the user should use an computed or overide the model's init method instead.
+
+The  work of lifecycle events can be acheived with a CP on a relevant property or kick off additional work when performing the operation.
+
```

Comment:
I think it would be nice with an actual example on how to go from the deprecated feature to using a computed property.

Reply:
Agreed, I'll work on that.

Reply:
Thank you so much!